spec/helpers/url_helpers_spec.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 RSpec.describe UrlHelpers do
-  let(:helper) do
+  def create_url_helper
```

Comment:
I think we could probably still call this `helper`.

```suggestion
  def helper
```